jme3-desktop/src/main/java/com/jme3/system/NativeLibrary.java

```diff
@@ -76,6 +79,31 @@ public boolean equals(Object obj) {
             return true;
         }
     }
+

```

Comment:
Could you add discriminative javadocs for those constructors documenting the constructor job and parameters usage and what [null] input means to the internal processing, please ?

Reply:
Done